Makefile

```
@@ -170,14 +161,11 @@ build/helm-operator-%-ppc64le-linux-gnu: GOARGS = GOOS=linux GOARCH=ppc64le
 build/helm-operator-%-s390x-linux-gnu: GOARGS = GOOS=linux GOARCH=s390x
 build/helm-operator-%-linux-gnu: GOARGS = GOOS=linux
 
-build/operator-%: $(SOURCES) ## Build the operator-sdk binary
-	$(Q)$(GOARGS) go build $(GO_BUILD_ARGS) -o $@ $(SDK_BUILD_PATH)
-
-build/ansible-%: $(SOURCES) ## Build the ansible-operator binary
-	$(Q)$(GOARGS) go build $(GO_BUILD_ARGS) -o $@ $(ANSIBLE_BUILD_PATH)
-
-build/helm-%: $(SOURCES) ## Build the helm-operator binary
-	$(Q)$(GOARGS) go build $(GO_BUILD_ARGS) -o $@ $(HELM_BUILD_PATH)
+build/%: $(SOURCES) ## Build the operator-sdk binary
```

I think we would need replace it as:
```suggestion
build-%: $(SOURCES) ## Build the operator-sdk binary
```